MFE/common/src/components/WizardForms/TemplateCreation/TemplateCreation.jsx

```diff
@@ -88,140 +90,181 @@ const TemplateCreation = ({
+          <Table size='small'>
+            <TableHead className={classes.tableHead}>
+              <TableRow>
+                <TableCell>
+                  <Tooltip
+                    classes={{ tooltip: classes.tooltip }}
+                    title={<Trans t={t} i18nKey='attrs:attrLabelHint' />}
+                    placement='top-start'
+                  >
+                    <Box display='flex' alignItems='center'>
+                      <strong>{t('attrs:attrLabel.attrLabel')}</strong>
+                      &nbsp;
+                      <span>*</span>
+                      &nbsp;
+                      <InfoOutlined fontSize='small' />
+                    </Box>
+                  </Tooltip>
+                </TableCell>
+
+                <TableCell>
+                  <Tooltip
+                    classes={{ tooltip: classes.tooltip }}
+                    title={<Trans t={t} i18nKey='attrs:attrTypeHint' />}
+                    placement='top-start'
+                  >
+                    <Box display='flex' alignItems='center'>
+                      <strong>{t('attrs:attrLabel.attrType')}</strong>
+                      &nbsp;
+                      <span>*</span>
+                      &nbsp;
+                      <InfoOutlined fontSize='small' />
+                    </Box>
+                  </Tooltip>
+                </TableCell>
+
+                <TableCell>
+                  <Tooltip
+                    classes={{ tooltip: classes.tooltip }}
+                    title={t('attrs:valueTypeHint')}
+                    placement='top-start'
+                  >
+                    <Box display='flex' alignItems='center'>
+                      <strong>{t('attrs:attrLabel.attrValueType')}</strong>
+                      &nbsp;
+                      <span>*</span>
+                      &nbsp;
+                      <InfoOutlined fontSize='small' />
+                    </Box>
+                  </Tooltip>
+                </TableCell>
+
+                <TableCell>
+                  <Tooltip
+                    classes={{ tooltip: classes.tooltip }}
+                    title={t('attrs:attrValueHint')}
+                    placement='top-start'
+                  >
+                    <Box display='flex' alignItems='center'>
+                      <strong>{t('attrs:attrLabel.attrValue')}</strong>
+                      &nbsp;
+                      <InfoOutlined fontSize='small' />
+                    </Box>
+                  </Tooltip>
+                </TableCell>
+
+                <TableCell />
+              </TableRow>
+            </TableHead>
+
+            <TableBody>
+              {attrs.map(({ id, label, type, valueType, staticValue }, index) => {
+                const isStaticAttr = type === TEMPLATE_ATTR_TYPES.STATIC.value;
+
+                const handleUpdateLabel = newLabel => {
+                  const reAttrLabel = /^[0-9a-zA-Z-_]+$/;
+
+                  if (newLabel.match(reAttrLabel)) {
```

Comment:
Maybe you can use `reAttrLabel.test(newLabel)` and use it directly with the `setAttrLabelError` function because it returns a boolean. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test